tests/integration/attendanceSheets.test.js

```diff
@@ -374,6 +375,31 @@ describe('ATTENDANCE SHEETS ROUTES - GET /attendancesheets', () => {
         expect(response.result.data.attendanceSheets.length).toEqual(1);
       });
 
+    it('should return a 403 if course company is not in holding and user has no vendor role', async () => {
```

Comment:
tu peux enlever `a` -> `should return 403`

Reply:
done